pkg/messaging/mqtt/pubsub.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const (
 	username = "mainflux-mqtt"
-	qos      = 2
+	qos      = 1 // At least once - Since NATS supports up to QoS 1.
```

Comment:
should be configurable with each profile or via env or some other way to pass as an argument